### PR TITLE
home handle events

### PR DESCRIPTION
**Pull request by victorbaxter**
## [Pull request title]
home handle events
### [Redmine link]
https://edu-redmine.framgia.vn/issues/7645

### CODE REVIEW

**Comment by victorbaxter** [comment]
ready

File: app/src/main/java/phuchh/com/music_60/ui/online/OnlineFragment.java [hunk 1]
```diff
@@ -45,7 +45,9 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
     public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
         initFragment();
-        mPresenter.getTopChart(Constant.ALL_MUSIC, Constant.LIMIT_DEFAULT, Constant.OFFSET_DEFAULT);
+        mPresenter.getTopChart(Constant.ALL_MUSIC,
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/phuchh/com/music_60/ui/online/OnlineFragment.java]: keep old code

File: app/src/main/java/phuchh/com/music_60/ui/online/OnlineFragment.java [hunk 2]
```diff
@@ -45,7 +45,9 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
...
+                Constant.LIMIT_DEFAULT, Constant.OFFSET_DEFAULT);
+        showGenre();
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/phuchh/com/music_60/ui/online/OnlineFragment.java]:
```suggestion
        showGenres();
```

File: app/src/main/java/phuchh/com/music_60/ui/online/OnlineFragment.java [hunk 3]
```diff
@@ -66,9 +80,12 @@ private void initFragment() {
         mPresenter = new OnlinePresenter(this,
                 TrackRepository.getInstance(TrackRemoteDataSource.getInstance(),
                         TrackLocalDataSource.getInstance()));
+    }
+
+    private void showGenre() {
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/phuchh/com/music_60/ui/online/OnlineFragment.java]:
```suggestion
    private void showGenres() {
```

**Comment by victorbaxter** [comment]
ready

**Pull request merged**